src/componentize.js

```diff
@@ -45,6 +45,7 @@ export async function componentize(jsSource, witWorld, opts) {
     worldName,
     disableFeatures = [],
     enableFeatures = [],
+    retainFetchEvent = false
```

Comment:
I wonder if it would make sense to integrate this into the `enableFeatures` model and call it `fetch-event`?

Reply:
Oooh! I like the idea of adding it via the `enableFeatures`. I will also add it to the readme.